lighty-modules/lighty-restconf-nb-community/src/main/java/io/lighty/modules/northbound/restconf/community/impl/CommunityRestConf.java

```diff
@@ -54,34 +45,27 @@ public class CommunityRestConf extends AbstractLightyModule {
     private final DOMMountPointService domMountPointService;
     private final DOMActionService domActionService;
     private final PortNumber webSocketPort;
-    private final JsonRestConfServiceType jsonRestconfServiceType;
     private final DOMSchemaService domSchemaService;
     private final int httpPort;
     private final InetAddress inetAddress;
     private final String restconfServletContextPath;
-    private RestconfImpl restconfImpl;
-    private BrokerFacade brokerFacade;
-    private ControllerContext controllerContext;
-    private RestconfProviderImpl restconfProvider;
     private Server jettyServer;
     private LightyServerBuilder lightyServerBuilder;
     private SchemaContextHandler schemaCtxHandler;
 
-    public CommunityRestConf(final DOMDataBroker domDataBroker, final DOMSchemaService schemaService,
-            final DOMRpcService domRpcService, final DOMActionService domActionService,
-            final DOMNotificationService domNotificationService, final DOMMountPointService domMountPointService,
-            final int webSocketPort, final JsonRestConfServiceType jsonRestconfServiceType,
-            final DOMSchemaService domSchemaService, final InetAddress inetAddress, final int httpPort,
-            final String restconfServletContextPath, final ExecutorService executorService,
-            final LightyServerBuilder serverBuilder) {
+    public CommunityRestConf(final DOMDataBroker domDataBroker, final DOMRpcService domRpcService,
+             final DOMActionService domActionService, final DOMNotificationService domNotificationService,
+             final DOMMountPointService domMountPointService, final int webSocketPort,
+             final DOMSchemaService domSchemaService, final InetAddress inetAddress,
+             final int httpPort, final String restconfServletContextPath,
+             final LightyServerBuilder serverBuilder) {
         this.domDataBroker = domDataBroker;
         this.domRpcService = domRpcService;
         this.domActionService = domActionService;
         this.domNotificationService = domNotificationService;
         this.domMountPointService = domMountPointService;
         this.lightyServerBuilder = serverBuilder;
         this.webSocketPort = new PortNumber(Uint16.valueOf(webSocketPort));
-        this.jsonRestconfServiceType = jsonRestconfServiceType;
         this.domSchemaService = domSchemaService;
         this.httpPort = httpPort;
         this.inetAddress = inetAddress;
```

Comment:
Please update documentation down here.

     /* In standard Oxygen ODL Restconf blueprints there are exposed two additional services:
           - jsonRestconfServiceDraft02 (for bierman implementation)
           - jsonRestconfServiceDraft02 (for bierman implementation)
           - jsonRestconfService (for RFC8040 implementation)
           - jsonRestconfService (for RFC8040 implementation)
           Those services are not expected to be required thus they are not here.
           Those services are not expected to be required thus they are not here.
         */
         
Also, search in ligty for "Bierman" or "Draft02" and update documentation.